core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -469,8 +474,8 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
       }
     }
 
-    def maybeFlushAndCommitOffsets() {
-      if (System.currentTimeMillis() - lastOffsetCommitMs > offsetCommitIntervalMs) {
+    def maybeFlushAndCommitOffsets(autoCommit: Boolean) {
+      if (!autoCommit || System.currentTimeMillis() - lastOffsetCommitMs > offsetCommitIntervalMs) {
```

Comment:
Thanks. I have fixed this as well. The patch has been tested with 2 clusters and 1 mirror maker.